lib/system/dyncalls.nim

```diff
@@ -167,14 +167,17 @@ elif defined(windows) or defined(dos):
 
 elif defined(genode):
 
-  proc nimUnloadLibrary(lib: LibHandle) {.
-    error: "nimUnloadLibrary not implemented".}
+  proc nimUnloadLibrary(lib: LibHandle) =
+    writeToStdErr("nimUnloadLibrary not implemented")
```

Comment:
quit is evil, refs https://forum.nim-lang.org/t/4042#25152
if you're turining this from CT error to RT error, don't use quit, use doAssert:
```
use doAssert false, "nimUnloadLibrary not implemented"
```

(in future we can add NotImplementedDefect for cases like this)

Reply:
Ok, when #17649 clears I can fix the rest of this file, I just copied code here from the nintendoswitch section.

Reply:
Or you can do it, I've rebased onto #17649.